lang/cs/build.props

```diff
@@ -75,6 +75,6 @@ under the License.
   <!-- Locations -->
   <PropertyGroup>
     <!--The root directory of the REEF source tree. -->
-    <REEF_Source_Folder>$([System.IO.Path]::GetFullPath($(SolutionDir)\..\..))</REEF_Source_Folder>
+    <REEF_Source_Folder>$([System.IO.Path]::GetFullPath($(MSBuildProjectDirectory)\..\..\..))</REEF_Source_Folder>
```

Comment:
It's reserved property, https://msdn.microsoft.com/en-us/library/ms164309.aspx : the absolute path of the directory where the project file is located.
It works better than `SolutionDir` because `SolutionDir` is always `..` regardless of the folder from which I call msbuild. This is ok if we build using sln, but to call msbuild from arbitrary folder (as required by the new project or just command line) we need something more stable (`SolutionDir` fails build on individual `.csproj` unless msbuild is called from the folder of this project).